weblate/trans/views/basic.py

```diff
@@ -315,6 +315,12 @@ def show_translation(request, project, component, lang):
         if test_component.can_add_new_language(user):
             other_translations.append(GhostTranslation(test_component, obj.language))
 
+    # Limit the number of other components displayed to 10, preferring untranslated ones
+    other_translations = sorted(
+        other_translations,
+        key=lambda t : t.stats.translated_percent
```

Comment:
I think it complains about this:

```suggestion
        key=lambda t : t.stats.translated_percent,
```

The best approach is install pre-commit hooks which will format the code, see https://docs.weblate.org/en/latest/contributing/code.html#coding-standard

Reply:
This is my first time committing using pre-commit and wow, this is great! I should have read the contributor docs thoroughly. Thanks a lot for your help!